<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowNuevaEmpresa</class>
 <widget class="QMainWindow" name="WindowNuevaEmpresa">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>466</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>779</width>
    <height>466</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>779</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/coffee.ico</normaloff>:/coffee.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; color:#ff0000;&quot;&gt;NOTA:  Todos los campos son obligatorios&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>451</width>
            <height>79</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="labelPrincipal">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>36</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>REGISTRO</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Administrador</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="lbNombre">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Nombre</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leNombre">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>211</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maxLength">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbApellido1">
               <property name="text">
                <string>Primer Apellido</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leApellido1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maxLength">
                <number>50</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="lbApellido2">
               <property name="text">
                <string>Segundo Apellido</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leApellido2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maxLength">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbDNI">
               <property name="text">
                <string>DNI</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leDNI">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>9</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="lbUsuario">
               <property name="text">
                <string>Nombre de Usuario</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leUsuario">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbCorreo">
               <property name="text">
                <string>Correo Electr√≥nico*</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leCorreo">
               <property name="maxLength">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="lbPass">
                 <property name="text">
                  <string>Contrase√±a*</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lePass">
                 <property name="maxLength">
                  <number>12</number>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="lbRepitePass">
                 <property name="text">
                  <string>Repite Contrase√±a*</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="leRepitePass">
                 <property name="maxLength">
                  <number>12</number>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Datos Empresa</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="lbNombreEmpresa">
               <property name="text">
                <string>Nombre Empresa</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leNombreEmpresa">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maxLength">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>CIF</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leCIF">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>191</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>19</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Direccion Empresa</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leDireccionEmpresa">
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Localidad</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leLocalidad">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>161</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>50</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Provincia</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leProvincia">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>181</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maxLength">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>C√≥digo Postal</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leCP">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>113</width>
                 <height>0</height>
                </size>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="maxLength">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Tel√©fono</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leTelefono1">
               <property name="maxLength">
                <number>9</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Correo Electr√≥nico</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leCorreoEmpresa">
               <property name="minimumSize">
                <size>
                 <width>331</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maxLength">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>52</width>
                 <height>13</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>47</width>
                 <height>13</height>
                </size>
               </property>
               <property name="text">
                <string>Tel√©fono2*</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leTelefono2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>129</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>129</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maxLength">
                <number>9</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>108</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="pbSalir">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Salir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbLimpiar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Limpiar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbCompletar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Completar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>108</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leNombre</tabstop>
  <tabstop>leApellido1</tabstop>
  <tabstop>leApellido2</tabstop>
  <tabstop>leDNI</tabstop>
  <tabstop>leUsuario</tabstop>
  <tabstop>leCorreo</tabstop>
  <tabstop>lePass</tabstop>
  <tabstop>leRepitePass</tabstop>
  <tabstop>leNombreEmpresa</tabstop>
  <tabstop>leCIF</tabstop>
  <tabstop>leDireccionEmpresa</tabstop>
  <tabstop>leLocalidad</tabstop>
  <tabstop>leProvincia</tabstop>
  <tabstop>leCP</tabstop>
  <tabstop>leTelefono1</tabstop>
  <tabstop>leCorreoEmpresa</tabstop>
  <tabstop>leTelefono2</tabstop>
  <tabstop>pbSalir</tabstop>
  <tabstop>pbLimpiar</tabstop>
  <tabstop>pbCompletar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
